src/constants/messages.js

```diff
@@ -0,0 +1,51 @@
+/*
+ Copyright (C) 2021 Aman Dwivedi (aman.dwivedi5@gmail.com)
```

Comment:
@caffeinated-pixels You can use your email here

Reply:
Hi. Is this something I need to add on my end, or can you do it on your end? Anyway, my email address is cakeisaliegaming@gmail.com.